src/components/overview/chart/index.tsx

```diff
@@ -0,0 +1,91 @@
+import CustomizedDot from '../customizedDot';
+import LinearGradients from '../linearGradients';
+import { colors } from '../../../theme';
+import { useState } from 'react';
+import {
+  Area,
+  AreaChart,
+  CartesianGrid,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { data, gradients, yDomain, yTicks } from './helper';
+import { tick, xTickPadding } from './styled';
+interface IEntry {
+  today: number;
+  hours: string;
+}
+const Chart = () => {
+  const currentDate = new Date();
+  const currentHour = currentDate.getHours();
+  const [isTodayHovered, setIsTodayHovered] = useState(false);
+  const [isYesterdayHovered, setIsYesterdayHovered] = useState(false);
+
+  const calculateTodayValue = (entry: IEntry, currentHour: number): number | null => {
```

Comment:
We can move it to helper

Reply:
fix: calculateTodayValue  moved to helper.ts